src/main/java/org/dependencytrack/resources/v1/ProjectResource.java

```diff
@@ -199,10 +202,12 @@ public Response getProjectsByClassifier(
             @ApiParam(value = "The classifier to query on", required = true)
             @PathParam("classifier") String classifierString,
             @ApiParam(value = "Optionally excludes inactive projects from being returned", required = false)
-            @QueryParam("excludeInactive") boolean excludeInactive) {
+            @QueryParam("excludeInactive") boolean excludeInactive,
+            @ApiParam(value = "Optionally excludes children projects from being returned", required = false)
```

Comment:
Was ist der unterschied von @ApiParam und @QueryParam?

Reply:
Der `@ApiParam` ist dazu da, um in Swagger einen Parameter (wie eben z.B. `@QueryParam`) detaillierter zu beschreiben.
Und `@QueryParam` ist dazu da, um einen Query-Parameter in eine Java-Methode zu übergeben und somit eine API-Request zu filtern.

Reply:
Ahh hast recht, jetzt erinner ich mich. passt so